aspnetcore/host-and-deploy/proxy-load-balancer.md

```diff
@@ -118,6 +119,23 @@ services.Configure<ForwardedHeadersOptions>(options =>
 | [OriginalHostHeaderName](/dotnet/api/microsoft.aspnetcore.builder.forwardedheadersoptions.originalhostheadername) | Use the header specified by this property instead of the one specified by [ForwardedHeadersDefaults.XOriginalHostHeaderName](/dotnet/api/microsoft.aspnetcore.httpoverrides.forwardedheadersdefaults.xoriginalhostheadername).<br><br>The default is `X-Original-Host`. |
 | [OriginalProtoHeaderName](/dotnet/api/microsoft.aspnetcore.builder.forwardedheadersoptions.originalprotoheadername) | Use the header specified by this property instead of the one specified by [ForwardedHeadersDefaults.XOriginalProtoHeaderName](/dotnet/api/microsoft.aspnetcore.httpoverrides.forwardedheadersdefaults.xoriginalprotoheadername).<br><br>The default is `X-Original-Proto`. |
 | [RequireHeaderSymmetry](/dotnet/api/microsoft.aspnetcore.builder.forwardedheadersoptions.requireheadersymmetry) | Require the number of header values to be in sync between the [ForwardedHeadersOptions.ForwardedHeaders](/dotnet/api/microsoft.aspnetcore.builder.forwardedheadersoptions.forwardedheaders) being processed.<br><br>The default in ASP.NET Core 1.x is `true`. The default in ASP.NET Core 2.0 or later is `false`. |
+::: moniker-end
+::: moniker range=">= aspnetcore-2.1"
+| Option | Description |
+| ------ | ----------- |
+| `AllowedHosts` | Restricts hosts by the `X-Forwarded-Host` header to the values provided. Port numbers must be excluded. If the list is empty, all hosts are allowed. A top level wildcard `*` allows all non-empty hosts. Subdomain wildcards are permitted but don't match the root (naked) domain [for example, `*.contoso.com` matches the subdomain `foo.contoso.com` but not the root (naked) domain `contoso.com`]. Unicode host names are allowed but are converted to [Punycode](https://tools.ietf.org/html/rfc3492) for matching. [IPv6 addresses](https://tools.ietf.org/html/rfc4291) must include bounding brackets and be in [conventional form](https://tools.ietf.org/html/rfc4291#section-2.2) (for example, `[ABCD:EF01:2345:6789:ABCD:EF01:2345:6789]`). Failure to restrict the allowed hosts may allow an attacker to spoof links generated by the service.<br><br>The default value is an empty [IList\<string>](/dotnet/api/system.collections.generic.ilist-1). |
```

Comment:
- top level --> top-level
- Delete the square braces around the following, and make it a sentence:
> [for example, `*.contoso.com` matches the subdomain `foo.contoso.com` but not the root (naked) domain `contoso.com`]

Reply:
It's one of the terms. Bare or apex are the others, but they're not as common.